src/clients/FlexbaseClient.Card.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Card } from '../models/Card/Card';
+import { Address } from '../models/Address/Address';
+import { CardOptions } from '../models/Card/CardOptions';
+import { FlexbaseClientBase } from './FlexbaseClient.Base';
+import { FlexbaseResponse } from '../models/FlexbaseResponse';
+
+interface QueryParameters {
+    searchTerm?: string;
+    status?: string;
+}
+
+interface CardsResponse extends FlexbaseResponse{
+    cards: Card[];
+}
+
+interface CardResponse extends FlexbaseResponse{
+    card: Card;
+}
+
+interface IssueCardForm {
+    cardType: string,
+    shipTo?: Address,
+    service?: string,
+}
+
+interface UpdateCardForm {
+    expensesTypes: {
+        amount: number,
+        groups: any,
```

Comment:
We need to strong type this. Is `groups` an array of objects? Perhaps, create a `Group` object and set this property to be of `Group[]` type.